config/config.exs

```diff
@@ -138,4 +138,5 @@ config :transport,
   nb_days_to_keep_validations: 60,
   join_our_slack_link: "https://join.slack.com/t/transportdatagouvfr/shared_invite/zt-2n1n92ye-sdGQ9SeMh5BkgseaIzV8kA",
   contact_email: "contact@transport.beta.gouv.fr",
+  security_email: "contact+security@transport.beta.gouv.fr",
```

Comment:
J'ai fait un test rapide avec gmail, ça semble ne pas fonctionner (`521 No Redirect Entry for this address`).

Reply:
Wow 😲  Merci d'avoir testé. Mais pourquoi donc une liste de diffusion OVH ne supporte pas le `+` ? Bref, j'ai changé dans le prochain commit.

Reply:
Ca, c'est une bonne question :-) Merci !